sdk/keyvault/azure-security-keyvault-keys/inc/azure/keyvault/keys/key_client_models.hpp

```diff
@@ -316,20 +316,53 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Keys {
     /**
      * @brief Gets the CKM_RSA_AES_KEY_WRAP algorithm.
      *
+     * @deprecated Use KeyEncryptionAlgorithm::CkmRsaAesKeyWrap.
      */
-    AZ_SECURITY_KEYVAULT_KEYS_DLLEXPORT static const KeyEncryptionAlgorithm CKM_RSA_AES_KEY_WRAP;
+    [[deprecated("Use "
```

Comment:
Is this sufficient to resolve the customer problem? 

Don't we have to provide a mechanism to disable the existing functions? Or to not include these definitions if they are already defined in the preprocessor?

Reply:
like this ?

Reply:
Yes, like this :).